aladdin/__init__.py

```diff
@@ -0,0 +1,2 @@
+VERSION = (1, 15, 6, 12)
```

Comment:
will we have something to automatically update this?

Reply:
suggestions welcome!